src/main/java/com/jfrog/ide/idea/scan/ScanManager.java

```diff
@@ -97,8 +95,8 @@ public void startScan() {
                     scanner.asyncScanAndUpdateResults();
                 }
                 executor.shutdown();
-                if (!executor.awaitTermination(SCAN_TIMEOUT_MINUTES, TimeUnit.MINUTES)) {
-                    logError(Logger.getInstance(), "Scan timeout of " + SCAN_TIMEOUT_MINUTES + " minutes elapsed. The scan is being canceled.", true);
+                if (!executor.awaitTermination(Long.MAX_VALUE, TimeUnit.MINUTES)) {
```

Comment:
I disagree with that approach. In a developer environment, it's better to end the scan after a few minutes. Plus, users don't get any feedback or indication about the ongoing scan during this prolonged period. But ultimately, it's up to you to decide.

Reply:
We will remove the timeout, add more logging, and add progress bars to enhance the user experience as discussed on Zoom